saleor/graphql/schema.graphql

```diff
@@ -2556,6 +2556,8 @@ type Mutation {
   shopSettingsTranslate(input: ShopSettingsTranslationInput!, languageCode: LanguageCodeEnum!): ShopSettingsTranslate
   shopAddressUpdate(input: AddressInput): ShopAddressUpdate
   shippingMethodChannelListingUpdate(id: ID!, input: ShippingMethodChannelListingInput!): ShippingMethodChannelListingUpdate
+  shippingMethodZipCodeCreate(input: ShippingZipCodeCreateInput!): ShippingZipCodeCreate
```

Comment:
It seems inconsistent with the rest of the mutations, where `id` is separate argument